addons/popochiu/editor/popups/setup/setup.gd

```diff
@@ -18,6 +18,8 @@ var es: EditorSettings = null
 
 var _selected_template: GUITemplateButton
 var _is_closing := false
+var _progress_lines := ["|", "/", "―", "\\", "―"]
```

Comment:
Leaving the comment here. It's not mandatory to merge this PR, but I would prefer a progress-bar to keep things consistent in the editor interface. An infinite one may do if you can't calculate the progress.

A last option is to have a progress bar updated in chunks depending on the phase of the phase of the copy/setup (like first 30% when you clean up the current GUI, next 30% when you copy the files over, last 30% when you do the final bits - just to explain, of course)

Reply:
Thanks! I'll add the progress bar with the chunks depending on the process phase.